builder/vsphere/clone/step_clone.go

```diff
@@ -67,7 +67,7 @@ type StepCloneVM struct {
 
 func (s *StepCloneVM) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
 	ui := state.Get("ui").(packer.Ui)
-	d := state.Get("driver").(*driver.Driver)
+	d := state.Get("driver").(*driver.VCenterDriver)
```

Comment:
Long term, we probably want to actually be casting to the Driver interface rather than the VCenterDriver struct implementation of that interface, right? That way we can put the mock into state and test it?

Reply:
Oh yeah!! This is one of the notes in the PR description. I want to do the change when I write the test related to it to be aware of where I'm changing it and making sure it works as expected.